katharsis-core/src/main/java/io/katharsis/resource/registry/responseRepository/RepositoryRequestSpecImpl.java

```diff
@@ -0,0 +1,183 @@
+package io.katharsis.resource.registry.responseRepository;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+import io.katharsis.dispatcher.controller.HttpMethod;
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.queryspec.DefaultQuerySpecConverter;
+import io.katharsis.queryspec.QuerySpec;
+import io.katharsis.queryspec.internal.QueryAdapter;
+import io.katharsis.queryspec.internal.QueryParamsAdapter;
+import io.katharsis.queryspec.internal.QuerySpecAdapter;
+import io.katharsis.repository.filter.RepositoryFilterContext;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.repository.filter.RepositoryFilterContext'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)